packages/features/users/components/UserTable/EditSheet/SheetFooterControls.tsx

```diff
@@ -0,0 +1,60 @@
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { SheetClose, Button } from "@calcom/ui";
+import { Pencil } from "@calcom/ui/components/icon";
+
+import { useEditMode } from "./store";
+
+function EditModeFooter() {
+  const { t } = useLocale();
+  const setEditMode = useEditMode((state) => state.setEditMode);
+  const isLoading = useEditMode((state) => state.mutationLoading);
+
+  return (
+    <>
+      <Button
+        color="secondary"
+        type="button"
+        className="justify-center md:w-1/5"
+        onClick={() => {
+          setEditMode(false);
+        }}>
+        {t("cancel")}
+      </Button>
+
+      <Button type="submit" className="w-full justify-center" form="edit-user-form" loading={isLoading}>
```

Comment:
Has a form="..."

as it points to a form that its not inside of. 

easiest way to do this. We do something like this in event-type-settings